CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
Tall order

Reply:
I've reverted this line, PR was in draft and I forgot to remove it since I was testing something locally with ++23 :)